.github/workflows/release.yml

```diff
@@ -0,0 +1,96 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+permissions:
+  contents: write
+  pull-requests: read
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 'v16.15.1'
+
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+        with:
+          registry: docker.elastic.co
+          secret: secret/observability-team/ci/docker-registry/prod
+          url: ${{ secrets.VAULT_ADDR }}
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+
+      - run: make -C .ci env
+
+      - run: make -C .ci dist
+
+      - run: make -C .ci push-docker
+
+      - uses: hashicorp/vault-action@v2.5.0
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/observability-team/ci/service-account/apm-aws-lambda access_key_id | AWS_ACCESS_KEY_ID ;
+            secret/observability-team/ci/service-account/apm-aws-lambda secret_access_key | AWS_SECRET_ACCESS_KEY
+
+      - run: |-
+          VERSION=$(cut -f 2 -d 'v' <<< $GITHUB_REF_NAME)
+          MAJOR_VERSION=$(cut -d '.' -f 1 <<< "$VERSION")
+          ELASTIC_LAYER_VERSION=$(sed 's#\.#-#g' <<< "$VERSION")
+          RELEASE_NOTES_URL="https://www.elastic.co/guide/en/apm/agent/nodejs/current/release-notes-${MAJOR_VERSION}.x.html#release-notes-${VERSION}" \
+          ELASTIC_LAYER_NAME=elastic-apm-node-ver-${ELASTIC_LAYER_VERSION} \
+          make -C .ci publish-in-all-aws-regions create-arn-file
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - run: make -C .ci github-release
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - uses: hashicorp/vault-action@v2.5.0
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/jenkins-ci/npmjs/elasticmachine token | NPMJS_TOKEN ;
+            totp/code/npmjs-elasticmachine code | TOTP_CODE
+
+      - run: |-
+          echo "//registry.npmjs.org/:_authToken=${{ env.NPMJS_TOKEN }}" > .npmrc
+          npm publish --otp=${{ env.TOTP_CODE }}
```

Comment:
For this and the other `- run: |-` multi-line commands, can we add a step name so the summary of a run is more readable?

![Screenshot 2023-03-29 at 11 06 43 AM](https://user-images.githubusercontent.com/46866/228629534-ce4fd0da-3b81-4c01-937b-0dd016e7b81d.png)